examples/remix-app-cloudflare/app/routes/_index.tsx

```diff
@@ -1,4 +1,4 @@
-import type {
```

Comment:
this `type` removal is unnecessary, I can revert it, I just included it to show that non-type imports from `@remix-run/cloudflare` now work (and they used not to: https://github.com/dario-piotrowicz/vite-environment-6.0.0-alpha-experimentations/issues/24)